Source/Eto/Drawing/Colors.cs

```diff
@@ -292,8 +292,19 @@ public static class Colors
 		// Gray colors
 		/// <summary>Gets a color with an ARGB value of #FFDCDCDC</summary>
 		public static Color Gainsboro { get { return Color.FromRgb(0xDCDCDC); } } // 220 220 220
+
+		// difference between Gray and Grey is very confusing
+		// some beleive that they are equal (while Gray being American and Grey British English)
+		// but while both may mean #808080, Grey sometimes refer to #D3D3D3 or "silver" (#C0C0C0)
+		// see http://www.greyorgray.com/ and System.Drawing.LightGray (no Grey or LightGrey)
+		// Grey should therfore not be present for not being exact and LightGray and LightGrey being aliases
+		// ...although LightGrey is actually wrong, but we keep it for compatibility, sanity and misspelling
+
```

Comment:
I will probably remove this comment. Although `LightGrey` was the only occurence of `Grey